File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/ServerTypes/RecriverSenderDB/RecriverSenderColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { RecriverSenderRow } from "./RecriverSenderRow";

export interface RecriverSenderColumns {
    Id: Column<RecriverSenderRow>;
    Name: Column<RecriverSenderRow>;
    IsDefault: Column<RecriverSenderRow>;
    CreatedDate: Column<RecriverSenderRow>;
    CreatorUserName: Column<RecriverSenderRow>;
    ModifiedDate: Column<RecriverSenderRow>;
    ModifiedUserName: Column<RecriverSenderRow>;
    TimeStamp: Column<RecriverSenderRow>;
}

export class RecriverSenderColumns extends ColumnsBase<RecriverSenderRow> {
    static readonly columnsKey = 'RecriverSenderDB.RecriverSender';
    static readonly Fields = fieldsProxy<RecriverSenderColumns>();
}

File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/RecriverSenderDB/RecriverSender/RecriverSenderColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace CorrespondenceSystem.RecriverSenderDB.Columns;

[ColumnsScript("RecriverSenderDB.RecriverSender")]
[BasedOnRow(typeof(RecriverSenderRow), CheckNames = true)]
public class RecriverSenderColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public Guid Id { get; set; }
    [EditLink]
    public string Name { get; set; }
    public bool IsDefault { get; set; }
    public DateTime CreatedDate { get; set; }
    public string CreatorUserName { get; set; }
    public DateTime ModifiedDate { get; set; }
    public string ModifiedUserName { get; set; }
    //public byte[] TimeStamp { get; set; }
}